grapher/facetChart/FacetChart.tsx

```diff
@@ -131,12 +133,12 @@ export class FacetChart
                 this.manager.yAxis!.facetAxisRange == FacetAxisRange.shared
                     ? {
                           max: sharedYDomain[1],
-                          min: sharedYDomain[0],
+                          min: yAxisMin ?? sharedYDomain[0],
```

Comment:
I think potentially the logic we want here is: 

```suggestion
                          min: Math.min(yAxisMin, sharedYDomain[0]),
```

Otherwise, I think, but am not sure, that the axes can be mismatched across charts. The `min` setting I think only extends the axes, it doesn't crop them (when data exists beyond the domain).

Reply:
It appears that the chart is already configured to not crop data, i.e. that `yAxis.min` is a guideline that can be ignored if needed to show the data.

Reply:
Right, but it doesn't seem straightforward what the behaviour should be?

Let's say we facet by entity, and one entity has range `[-10, 10]` and all the others have `[0,10]`, and `yAxisMin = 0`. 

Should the shared axis be `[-10, 10]` or `[0, 10]`? If we go for the latter (I think what the PR currently does), one chart will have an axis not matching the others.

Reply:
Ah, good call. Given the graph's flexible use of `yAxis.min` and `yAxis.max`, we need to cap them the way you're describing, otherwise the y axis will not be the same across facets. Good catch! 🙌